spec/mailers/candidate_mailer_spec.rb

```diff
@@ -109,6 +109,79 @@
     end
   end
 
+  describe 'Offer X day mailers' do
+    let(:offer) do
+      build_stubbed(:application_choice, :offered,
+                    sent_to_provider_at: Time.zone.today,
+                    application_form: build_stubbed(:application_form, first_name: 'Fred'),
+                    course_option:)
+    end
+    let(:course_option) do
+      build_stubbed(:course_option, course: build_stubbed(:course,
+                                                          name: 'Applied Science (Psychology)',
+                                                          code: '3TT5', provider:))
+    end
+    let(:provider) { build_stubbed(:provider, name: 'Brighthurst Technical College') }
+    let(:application_choices) { [offer] }
+
+    delegate :name, to: :provider, prefix: true
```

Comment:
What's this for?

Reply:
That's a careless copy/paste mistake, removed!